docker/Dockerfile-postgis

```diff
@@ -0,0 +1,9 @@
+ARG POSTGRES_VERSION
+FROM postgres:${POSTGRES_VERSION}
```

Comment:
I suggest using https://github.com/appropriate/docker-postgis

Reply:
I do like the idea of using that. Initially I wanted to try and only use official images, but I'm not sure how much point there is to that.